spec/features/test_result_spec.rb

```diff
@@ -49,6 +49,10 @@ def clean_test_result(test_result)
     test_result["test_result"]["runtime"] = "<cleaned for test>"
     test_result["test_result"]["started_at"] = "<cleaned for test>"
     test_result["test_result"]["finished_at"] = "<cleaned for test>"
+    test_result["test_result"]["status"]["backtrace"]&.map! do |line|
+      line.gsub!(/:\d+:/, ":<line-number-cleaned-for-tests>:")
+    end
```

Comment:
Is this mainly to avoid frequent updates to the snapshot files?

Reply:
yes